tests/tests/test-block/test-block-gas.ts

```diff
@@ -1,13 +1,13 @@
 import { expect } from "chai";
-import { describeDevMoonbeam } from "../../util/setup-dev-tests";
+import { describeDevMoonbeam, describeDevMoonbeamAllEthTxTypes } from "../../util/setup-dev-tests";
```

Comment:
a lot of those describeDevMoonbeam imports could be removed ut its not very important

Reply:
There's a skipped test still using it. We can leave it there by now.